### PR TITLE
Сlarify React Native runtime support

### PR DESCRIPTION
**Pull request by farwayer**
Notes about iOS 9, RN 0.59 and new Hermes runtime.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -155,7 +155,7 @@ MobX 5 is the first MobX version fully leveraging Proxies. This has two big adva
 
 ### The system requirements to run MobX has been upped
 
-* MobX 5 can only be used on environments that support `Proxies`. In practice this means, no Internet Explorer (Edge is fine). No nodejs < 6. React Native on Android only when JavaScript core is [upgraded](https://github.com/react-community/jsc-android-buildscripts#how-to-use-it-with-my-react-native-app). All modern browsers are supported.
+* MobX 5 can only be used on environments that support `Proxies`. In practice this means, no Internet Explorer (Edge is fine). No nodejs < 6. React Native supported from iOS 10, on Android from RN 0.59 (or with manual JavaScript core [upgrade](https://github.com/react-community/jsc-android-buildscripts#how-to-use-it-with-my-react-native-app)), new [Hermes runtime](https://github.com/facebook/hermes/issues/28) is not supported. All modern browsers are supported.
```
Comment (FredyC) [inline CHANGELOG.md]: I think it might be better to separate out all RN related information to an additional bullet point. Same for the update in readme.

Comment (farwayer) [reply CHANGELOG.md]: Is
```
In practice this means:
- no Internet Explorer (Edge is fine)
- nodejs >= 6
React Native:
- iOS >= 10
- Android from RN 0.59 (or with manual JavaScript core [upgrade](https://github.com/react-community/jsc-android-buildscripts#how-to-use-it-with-my-react-native-app))
- new Hermes runtime is [not supported](https://github.com/facebook/hermes/issues/28)
```
will be ok?

Comment (FredyC) [reply CHANGELOG.md]: Sure, sounds reasonable :) I would drop a word `new` as it's not relevant and might be one day "old" :)

File: README.md [hunk 2]
```diff
@@ -48,7 +48,14 @@ _Tip: Consider using the faster and smaller ES6 build if targetting a modern env
...
 
--   MobX >=5 runs on any browser with [ES6 proxy support](https://kangax.github.io/compat-table/es6/#test-Proxy). It will throw an error on startup on older environments such as IE11, Node.js <6 or React Native Android on old JavaScriptCore [how-to-upgrade](https://github.com/react-community/jsc-android-buildscripts#how-to-use-it-with-my-react-native-app).
+-   MobX >=5 runs on any browser with [ES6 proxy support](https://kangax.github.io/compat-table/es6/#test-Proxy). In practice this means:
+    - no Internet Explorer (Edge is fine)
+    - Node.js >= 6  
+React Native:
+    - iOS >= 10
+    - Android from RN 0.59 (or with manual JavaScript core [upgrade](https://github.com/react-community/jsc-android-buildscripts#how-to-use-it-with-my-react-native-app))
+    - Hermes runtime is [not supported](https://github.com/facebook/hermes/issues/28)  
+All modern browsers are supported.
```
Comment (FredyC) [inline README.md]:
```suggestion
    - All modern browsers are supported.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -155,7 +155,14 @@ MobX 5 is the first MobX version fully leveraging Proxies. This has two big adva
 
 ### The system requirements to run MobX has been upped
 
-* MobX 5 can only be used on environments that support `Proxies`. In practice this means, no Internet Explorer (Edge is fine). No nodejs < 6. React Native on Android only when JavaScript core is [upgraded](https://github.com/react-community/jsc-android-buildscripts#how-to-use-it-with-my-react-native-app). All modern browsers are supported.
+* MobX 5 can only be used on environments that support `Proxies`. In practice this means:
```
Comment (FredyC) [inline CHANGELOG.md]: Instead of duplicating this in changelog, it might be better to just link to README.

File: README.md (hunk 2 above)
Comment (mweststrate) [reply README.md]: Applied

File: CHANGELOG.md (hunk 3 above)
Comment (mweststrate) [reply CHANGELOG.md]: Updated

File: README.md (hunk 2 above)
Comment (farwayer) [reply README.md]: Sorry was unavailable. Was this expected change that you removed the sub-items?

*Browser support* section looks like
2019-07-29-125915_892x141_scrot

In my branch 
2019-07-29-130248_764x196_scrot

Comment (FredyC) [reply README.md]: Well, it's mostly the same, just restructured differently. All the information is there.

**Pull request merged**